DependencyInjection/FOSRestExtension.php

```diff
@@ -171,6 +171,10 @@ private function loadFormatListener(array $config, XmlFileLoader $loader, Contai
                     'fos_rest.format_listener.media_type.version_regex',
                     $config['format_listener']['media_type']['version_regex']
                 );
+                $container->setParameter(
+                    'fos_rest.format_listener.media_type.default_version',
+                    $config['format_listener']['media_type']['default_version']
+                );
```

Comment:
Probably, though I think 2.0 would be good release to clean them up (another advantage is that you reduce the size of the compiled container).